src/components/questions/answersList.js

```diff
@@ -10,13 +10,18 @@ class AnswerList extends React.Component{
     };
   }
 
+  makeAnswerArray(answerObj){
+
+  }
+
   render(){
     return (
-    <div>
       <div>
-        AnswerList (map answers to Answerlist)
-      </div>
-      <Answer/>
+      {
+      Object.keys(this.props.answers)[0] ? (Object.keys(this.props.answers).map( (key, index) => {
+        return <Answer key={index} id={this.props.answers[key].id} body={this.props.answers[key].body} date={this.props.answers[key].date} answerer_name= {this.props.answers[key].answerer_name} helpfulness={this.props.answers[key].helpfulness} photos={this.props.answers[key].photos}/>
+      })) : <div>{" "}</div>
+      }
```

Comment:
if this is just a list of keys consider using a set

Reply:
its actually just a work around for using map with an object, i will comment in whats going on when we're not on this time crunch for our upcoming demo